eoscostarica/eos-local
# bug(locales): remove old languages files and fix scripts

# Changes
1. Add missing scripts to copy  languages into public folder
2. Remove old language files
3. Change copy npm module

# Steps to test
1. Run the project (dev or build mode)
2. Double check the language folder have been moved and rename to public/locales
3. Check if translation are working in the app
